### PR TITLE
added Apache superset charts

### PR DESCRIPTION
**Pull request by alitari**
Helm charts for  [Apache superset](https://superset.incubator.apache.org/)

### CODE REVIEW

File: incubator/superset/values.yaml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+tolerations: []
+
+## Affinity and anti-affinity
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
```
Comment (unguiculus) [inline incubator/superset/values.yaml]: Don't specify defaults for resources. Comment them out.

File: incubator/superset/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+version: 0.1.0
+appVersion: "0.23.3"
+keywords:
+- bi
+home: https://github.com/apache/incubator-superset
+icon: https://superset.incubator.apache.org/_images/s.png
+sources:
+- https://github.com/apache/incubator-superset
+maintainers:
+- name: Alexander Krieg
```
Comment (unguiculus) [inline incubator/superset/Chart.yaml]: Use Github username

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by alitari** [comment]
Something went wrong with my repo, so i made a new pr #4481

**Pull request closed without merging**